ccmlib/scylla_node.py

```diff
@@ -567,24 +568,39 @@ def import_bin_files(self):
                                  'tools', 'bin'))
         for name in files:
             self.hard_link_or_copy(os.path.join(self.get_install_dir(),
-                                                'resources', 'cassandra',
+                                                tool_dir,
```

Comment:
`tool_dir`'s definition includes `self.get_install_dir()` in the default case.
Seems odd it's join()ed here this way...  Shouldn't it be same as above?
```
-            self.hard_link_or_copy(os.path.join(self.get_install_dir(),
-                                                'resources', 'cassandra',
+            self.hard_link_or_copy(os.path.join(tool_dir,
```

Reply:
you are correct, I have used those specific tools, hence I've missed it